src/components/Popups/TransactionPopup.tsx

```diff
@@ -17,25 +19,41 @@ export default function TransactionPopup({
   hash,
   success,
   summary,
+  swapProtocol,
 }: {
   hash: string
   success?: boolean
   summary?: string
+  swapProtocol?: SwapProtocol
 }) {
   const { chainId } = useActiveWeb3React()
 
   const theme = useTheme()
 
+  const isGnosisProtocolHash = swapProtocol === SwapProtocol.COW
+
+  const explorer = 'View on ' + (isGnosisProtocolHash ? 'Gnosis Protocol Explorer' : 'block explorer')
+
+  const body = chainId && (
```

Comment:
I think _body_ should be uppercase since it's a component. 

Also don't think body is a clear name.

Reply:
Done.

Reply:
Done.